### PR TITLE
Improve readability of verify text

### PR DESCRIPTION
**Pull request by MorrisJobke**
Makes a bit more sense when reading it.

### CODE REVIEW

File: settings/Controller/UsersController.php [hunk 1]
```diff
@@ -611,15 +611,15 @@ public function getVerificationCode($account, $onlyVerificationCode) {
...
+				$msg = $this->l10n->t('In order to verify your Twitter account, post following tweet on Twitter (please make sure to post it without any line breaks):');
 				$code = $codeMd5;
 				$type = AccountManager::PROPERTY_TWITTER;
 				$data = $accountData[AccountManager::PROPERTY_TWITTER]['value'];
 				$accountData[AccountManager::PROPERTY_TWITTER]['signature'] = $signature;
 				break;
 			case 'verify-website':
 				$accountData[AccountManager::PROPERTY_WEBSITE]['verified'] = AccountManager::VERIFICATION_IN_PROGRESS;
-				$msg = $this->l10n->t('In order to verify your Website store following content in your web-root at \'.well-known/CloudIdVerificationCode.txt\' (please make sure that the complete text is in one line):');
+				$msg = $this->l10n->t('In order to verify your Website, store following content in your web-root at \'.well-known/CloudIdVerificationCode.txt\' (please make sure that the complete text is in one line):');
```
Comment (ChristophWurst) [inline settings/Controller/UsersController.php]: "store following content" -> "store the following content"?

Comment (ChristophWurst) [inline settings/Controller/UsersController.php]: also "(please make sure that the complete text is in one line)" is inconsistent with "(please make sure to post it without any line breaks)", which should mean the same thing right?

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: settings/Controller/UsersController.php (hunk 1 above)
Comment (MorrisJobke) [inline settings/Controller/UsersController.php]: I would leave this. I fixed the article.

Comment (ChristophWurst) [inline settings/Controller/UsersController.php]: Fine by me :)

**Review by ChristophWurst — APPROVED** [review]

**Review by MariusBluem — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

**Pull request merged**